docs/managing_deis/running-stateless-cluster.rst

```diff
@@ -0,0 +1,214 @@
+to Deis via the ``git push`` workflow. It pushes to registry to store releases,
+so it will require no changes.
+
+Store
+^^^^^
+
+The :ref:`store` components implement Ceph itself. In a Ceph-less cluster, we
+will skip the installation and starting of these components. We will also
+mock a fake ``deis-store-volume`` component so that the controller and logger
+are happy.
+
+Deploying the cluster
+---------------------
+
+This guide assumes a typical deployment on AWS by following the :ref:`deis_on_aws`
+guide.
+
+Deploy an AWS cluster
+^^^^^^^^^^^^^^^^^^^^^
+
+Follow the :ref:`deis_on_aws` installation documentation through the "Configure
+DNS" portion.
+
+Configure log shipping
+^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`logspout` component must be configured to ship logs to somewhere other
+than the :ref:`logger` component.
+
+.. code-block:: console
+
+    $ HOST=logs.somewhere.com
+    $ PORT=98765
+    $ deisctl config logs set host=${HOST} port=${PORT}
+
+Configure registry
+^^^^^^^^^^^^^^^^^^
+
+The :ref:`registry` component won't start until it's configured with an S3 store.
+
+.. code-block:: console
+
+    $ BUCKET=MYS3BUCKET
+    $ AWS_ACCESS_KEY=something
+    $ AWS_SECRET_KEY=something
+    $ deisctl config registry set s3bucket=${BUCKET} \
+                                  bucketName=${BUCKET} \
+                                  s3accessKey=${AWS_ACCESS_KEY} \
+                                  s3secretKey=${AWS_SECRET_KEY} \
+                                  s3region=eu-west-1 \
+                                  s3path=/ \
+                                  s3encrypt=false \
+                                  s3secure=false
+    $ deisctl config store set gateway/accessKey=${AWS_ACCESS_KEY} \
+                               gateway/secretKey=${AWS_SECRET_KEY} \
+                               gateway/host=s3.amazonaws.com \
+                               gateway/port=80
+
+Configure database settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since we won't be running the :ref:`database`, we need to configure these settings
+so the controller knows where to connect.
+
+.. code-block:: console
+
+    $ HOST=something.rds.amazonaws.com
+    $ DB_USER=deis
+    $ DB_PASS=somethingsomething
+    $ DATABASE=deis
+    $ deisctl config database set engine=postgresql_psycopg2 \
+                                  host=${HOST} \
+                                  port=5432 \
+                                  name=${DATABASE } \
+                                  user=${DB_USER} \
+                                  password=${DB_PASS}
+
+Fake deis-store-volume
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``deis-store-volume`` component mounts the shared Ceph filesystem used by
+both the :ref:`logger` and :ref:`controller`. For the controller to run, it needs
+to think that the log directory exists, so we create a fake ``deis-store-volume``
+component.
+
+This code snippet is intended to be run on a CoreOS host because it uses
+``fleetctl`` directly.
```

Comment:
Would it be possible to make deisctl install this for us? To remove a manual step.